src/spatial_ref/srs.rs

```diff
@@ -379,7 +370,7 @@ impl SpatialRef {
     }
 
     pub fn axis_name(&self, target_key: &str, axis: i32) -> Result<String> {
-        // See get_axis_orientation
+        // See axis_orientation
         let c_ptr = unsafe {
```

Comment:
axis names are assumed to be static, so no need to free it?

Reply:
Comes from `OGR_SRSNode`.